### PR TITLE
tests: Migrate to assertj

### PR DESCRIPTION
**Pull request by matriv**

### CODE REVIEW

File: server/src/test/java/io/crate/integrationtests/TransportSQLActionClassLifecycleTest.java [hunk 1]
```diff
@@ -189,8 +197,8 @@ public void testGlobalAggregateNullRowWithoutMatchingRows() throws Exception {
         SQLResponse response = execute(
             "select sum(age), avg(age) from characters where characters.age > 112");
         assertThat(response.rowCount()).isEqualTo(1);
-        assertThat(response.rows()[0][0]).isNull();
-        assertThat(response.rows()[0][1]).isNull();
+        assertNull(response.rows()[0][0]);
+        assertNull(response.rows()[0][1]);
```
Comment (mfussenegger) [inline server/src/test/java/io/crate/integrationtests/TransportSQLActionClassLifecycleTest.java:201]: I'm confused, this is the opposite of what we want?

Comment (matriv) [reply server/src/test/java/io/crate/integrationtests/TransportSQLActionClassLifecycleTest.java:201]: Apologies, I somehow reverted this file to a version before a previous PR, I'm reverting, thx for catching!

**Comment by matriv** [comment]
FYI, I've added a few more files to a separate commit (https://github.com/crate/crate/pull/16267/commits/33b73be164d44505d1d7638bbd3fe3fd858b42d3)

**Review by mfussenegger — APPROVED** [review]
:+1:

**Pull request merged**